hub/core/chunk/sample_compressed_chunk.py

```diff
@@ -86,6 +84,8 @@ def read_sample(  # type: ignore
                     f"subfile,,start,{header_size + sb},end,{header_size + eb},,:"
                     + bytes(buffer[:-4]).decode("utf-8")
                 )
+                if not decompress:
```

Comment:
what does decompress=False along with stream even mean? could you elaborate?

Reply:
It gets the video url only without decompressing the video itself